ADR/ADR037-implement-review-apps.md

```diff
@@ -0,0 +1,187 @@
+# ADD037: Implement review apps
+
+Date: 2024-11-25
+
+## Status
+Proposed
+
+## Context
+
+### What are review apps?
+Review apps are short-lived deployments of applications, based on a branch other than the main branch. They are often created automatically in response to a new pull request, and are kept up to date with the branch automatically. They are usually destroyed when the pull request is closed.
+
+They exist to make it easy to create a live version of an application for review.
+
+In the GOV.UK Forms context, deploying a review copy of one app will inherently mean deploying a short-lived copy of another, because we operate a microservice architecture.
+
+### Why do we want them
```

Comment:
Non-urgent suggestion (feel free to ignore).

I wonder if it's worth highlighting things review apps won't be able to do/ Are there any changes which technical and non-technical teammates won't be able to do with this proposal? 

Something like, review apps will be limited to a test data set and won't interact with production data. Or perhaps, review apps won't be able to test third party integrations (e.g., Notify or something).

Reply:
That's a good idea. I'll add something.